### PR TITLE
[mailhog] Add imagePullSecrets to chart

### PR DESCRIPTION
**Pull request by glothriel**
Due to this whole Docker quota drama we are mirroring all images to private registry, so we need this setting configurable.

Signed-off-by: Konstanty Karagiorgis <k.karagiorgis@piwik.pro>

### CODE REVIEW

**Comment by glothriel** [comment]
Sorry @unguiculus for pinging, but It blocks the progress of what I'm currently doing - could You review this PR?

File: charts/mailhog/templates/deployment.yaml [hunk 1]
```diff
@@ -30,6 +30,10 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.image.pullSecrets }}
```
Comment (unguiculus) [inline charts/mailhog/templates/deployment.yaml:33]: Don't group it under `image`.
```suggestion
      {{- with .Values.imagePullSecrets }}
```

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: charts/mailhog/templates/deployment.yaml (hunk 1 above)
Comment (glothriel) [reply charts/mailhog/templates/deployment.yaml]: Done. At this point, after configuring this setting in ~20 different helm charts I'm confident to say that I've seen almost every possible approach to setting this value :D

Comment (unguiculus) [reply charts/mailhog/templates/deployment.yaml]: Yes, agreed. But this way is consistent with how it is in the manifest.

**Review by unguiculus — APPROVED** [review]

**Comment by exaucae** [comment]
Bump: can't  we just apply this to the keycloak theme?

**Pull request merged**